test/components/ColumnCard.nuxt.test.ts

```diff
@@ -0,0 +1,35 @@
+import { screen } from '@testing-library/vue'
+import { renderSuspended } from '@nuxt/test-utils/runtime'
+import { ColumnCard } from '#components'
+import { taskTruncatedIdLength } from '~/constants/tasks'
+
+describe('<ColumnCard />', () => {
+  it('should render the title (link to a new tab), status, id (truncated), and labels', async () => {
```

Comment:
Would be good if we would cover edge cases for this component too. For example every time there's a branching e.g. this [`v-if`](https://github.com/cytechmobile/radicle-planning-boards/pull/73/files#diff-4bdcf814fcd7c205d380617026790b5055c0e9a6a7dc4f49973cb0fe62b00728R41)

But for this specific case, I don't see it as a hard requirement. Up to you.

Reply:
I agree that edge cases should be covered by tests. But in this specific case I don't think it would add any value.

It would be the same as testing that the title doesn't render when an empty string is passed. Pretty redundant.